setup.py

```diff
@@ -41,29 +24,43 @@ def load_requirements(path_dir=PATH_ROOT, filename="base.txt", comment_char="#")
     return reqs
 
 
+def load_version() -> str:
+    """Load version from the lightly/__init__.py file.
+
+    Note: We do not want to get the version with `lightly.__version__` because it would
+    require importing `lightly`. We do not want to import lightly during setup/install
+    because it has side effects and it expects dependencies to installed. This is
+    however only the case after the package has been installed.
+    """
+    version_filepath = _PATH_ROOT / "lightly" / "__init__.py"
+    with version_filepath.open() as file:
+        for line in file.readlines():
+            if line.startswith("__version__"):
+                version = line.split("=")[-1].strip().strip('"')
+                return version
+    raise RuntimeError("Unable to find version string in '{version_filepath}'.")
+
+
 if __name__ == "__main__":
     name = "lightly"
-    version = lightly.__version__
-    description = lightly.__doc__
-
-    author = "Philipp Wirth & Igor Susmelj"
-    author_email = "philipp@lightly.ai"
+    version = load_version()
+    author = "Lightly Team"
+    author_email = "team@lightly.ai"
     description = "A deep learning package for self-supervised learning"
+    long_description = (_PATH_ROOT / "README.md").read_text()
 
     entry_points = {
         "console_scripts": [
             "lightly-crop = lightly.cli.crop_cli:entry",
-            "lightly-train = lightly.cli.train_cli:entry",
+            "lightly-download = lightly.cli.download_cli:entry",
             "lightly-embed = lightly.cli.embed_cli:entry",
             "lightly-magic = lightly.cli.lightly_cli:entry",
-            "lightly-download = lightly.cli.download_cli:entry",
-            "lightly-version = lightly.cli.version_cli:entry",
             "lightly-serve = lightly.cli.serve_cli:entry",
+            "lightly-train = lightly.cli.train_cli:entry",
+            "lightly-version = lightly.cli.version_cli:entry",
         ]
     }
 
-    long_description = load_description()
-
```

Comment:
We should list the packages below automatically (line 82, I can't comment there)